cpp/oneapi/dal/test/ccl_communicator.cpp

```diff
@@ -143,16 +190,52 @@ TEST_M(ccl_comm_test, "allreduce") {
         test_allreduce(buffer, count);
     }
 
+#ifdef ONEDAL_DATA_PARALLEL
     SECTION("device") {
         test_allreduce_on_device(buffer, count);
     }
+#endif
 
     const std::int64_t rank_count = get_new_comm().get_rank_count();
     for (std::int64_t i = 0; i < count; i++) {
         REQUIRE(buffer[i] == float(rank_count));
     }
 }
 
+// TEST_M(ccl_comm_test, "allreduce single value") {
+//     constexpr std::int64_t count = 1;
+
+//     float buffer[count];
+//     buffer[0] = 1.0f;
+
+//     SECTION("host") {
+//         test_allreduce(buffer, count);
+//     }
+
+//     SECTION("device") {
```

Comment:
Single value methods aren't expected to work on device - please remove "device" section. This test should work on host - please uncomment it.